tests/test_search_aggregates.py

```diff
@@ -409,7 +409,21 @@ def test_add_derived_columns(exploded_data_for_derived_cols,
 def test_basic_aggregation(main_summary,
                            expected_search_dashboard_data,
                            df_equals):
+    # it is too complicated to test the contents of the client count hll
+    # (it's essentially an inscrutable binary blob), so we'll do a basic
+    # aggregation to test its contents, then drop it from the table for
+    # the purposes of comparison
+
     actual = search_aggregates(main_summary)
+
+    # hll validation
+    assert ('client_count_hll', 'binary') in actual.dtypes
+    hll_count = actual.groupBy('client_count_hll').agg(
```

Comment:
I'm having a hard time understanding how to interpret a groupBy on an HLL binary variable. Can we either groupBy some categorical variable (like country) or just get an aggregate count of clients (i.e. 3)?